src/correction.cc

```diff
@@ -467,6 +486,9 @@ Binning::Binning(const JSONObject& json, const Correction& context)
   else if ( flowbehavior == "error" ) {
     flow_ = _FlowBehavior::error;
   }
+  else if ( flowbehavior == "wrap" ) {
```

Comment:
Given the number of options here, and the duplicated code in `MultiBinning::MultiBinning` can you factor this out into a free function `parse_flowbehavior`?

Reply:
Sure, I added a commit with that code factored out to a separate function.